report.md

```diff
@@ -126,6 +127,42 @@ https://github.com/transmissions11/solmate/blob/main/src/utils/SafeTransferLib.s
 `@dev Note that none of the functions in this library check that a token has code at all! That responsibility is delegated to the caller`
 
 
+- Found in src/ArbitraryTransferFrom.sol [Line: 20](tests/contract-playground/src/ArbitraryTransferFrom.sol#L20)
```

Comment:
These shouldn't be caught by the solmate detector.

Reply:
This happens because the somate detector first checks if there are ANY solmate imports across the whole project, then finds all references to:

```
if member_access.member_name == "safeTransfer"
                || member_access.member_name == "safeTransferFrom"
                || member_access.member_name == "safeApprove"
```

Solutions:
- Check for solmate imports and recommend against -> This would alter the solmate detector to find import statements, and not the actual references -> some false positives
- Check that the references to these functions only occur in source_units where the solmate dependency is imported -> some false negatives (worse)